src/shared/validation.ts

```diff
@@ -260,6 +260,7 @@ const mapObjectBaseSharedValidators = {
   position: isRRPoint,
   rotation: z.number(),
   playerId: isRRID<RRPlayerID>(),
+  roughness: z.number(),
```

Comment:
Not every map object needs a roughness value (i.e. images and text), so maybe we need a specific shared validators for `roughMapObjects`.

Reply:
True - adding the roughness property to `mapObjectDrawingSharedValidators` instead of `mapObjectBaseSharedValidators` improves this a little bit, since it excludes tokens, but it still includes images and text. On the other hand, images appear to currently also have a `color`, which we don't use.
I'm fine either way.